src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2852,6 +2852,21 @@ def update_http_proxy_config(self, mc: ManagedCluster) -> ManagedCluster:
         mc.http_proxy_config = self.context.get_http_proxy_config()
         return mc
 
+    def set_up_kube_proxy_config(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
Better name it something like `update_kube_proxy_config` to distinguish `set_up_kube_proxy_config` in `AKSPreviewManagedClusterCreateDecorator`.

Also, need to call this function in `update_mc_profile_preview`, otherwise it won't work?

Reply:
fixed, thanks